Jenkinsfile

```diff
@@ -0,0 +1,29 @@
+#!groovy
+
+node {
+  checkout scm
+
+  def dockerRepoName = 'zooniverse/AnnoTate'
+  def dockerImageName = "${dockerRepoName}:${BRANCH_NAME}"
+  def newImage = null
+
+  stage('Build app') {
+    sh 'npm install'
+    sh 'npm run build'
+  }
+
+  stage('Build Docker image') {
+    newImage = docker.build(dockerImageName)
+    newImage.push()
```

Comment:
Should we remove the push if we don't want to re-use these images? I.e. build a local image and use it to run the install / build steps.
https://jenkins.io/doc/book/pipeline/docker/#building-containers
```
def customImage = docker.build("my-image:${env.BUILD_ID}")
    customImage.inside {
        sh 'make test'
    }
```